### DISCUSSION TITLE
How to Structure Data Returned by the Supabase API for Game Results?

### DISCUSSION THREAD

**Discussion by caiocaezar** [supabase/supabase#21931] [Questions]
Hello everyone,

I’m working on a project that involves fetching data about game results from a sports league using the Supabase API. However, I’m facing an issue: the result of my query presents all game information in a single column, making it challenging for analysis.

Here’s the code I’m using:

```
with
  rodadas_json as (
    select
      content::json -> 'response' as rodadas
    from
      http (
        (
          'GET',
          'https://api-football-v1.p.rapidapi.com/v3/fixtures?league=475&season=2024',
          array[
            http_header (
              'X-RapidAPI-Key',
              'API_KEY'
            ),
            http_header (
              'X-RapidAPI-Host',
              'api-football-v1.p.rapidapi.com'
            )
          ],
          null,
          null
        )::http_request
      )
  )
select
  json_array_elements_text(rodadas_json.rodadas) as rodada
from
  rodadas_json;
```
How can I structure the returned data so that each field (such as game ID, referee, date, score, etc.) is displayed in separate columns? I’m using Supabase, and I’m wondering if anyone has encountered a similar situation and can help me solve this problem.

The image of the actual result:
![Captura de tela 2024-03-10 215525](https://github.com/supabase/supabase/assets/11754141/7f605ce0-a130-4576-9ec4-31ec3b69421a)

I appreciate your assistance in advance!

**Comment by emerson-proenca** [comment]
How did you solve it? I'm facing the same problem
